src/common/outputService/outputService.ts

```diff
@@ -20,6 +24,16 @@ export interface OutputService {
    * Prints a summary of the operation being done
    */
   printOpSummary(): void;
+
+  /**
+   * Displays a table
+   */
+  displayTable(
+    rows: any[],
+    title: string,
+    columns: CliUx.Table.table.Columns<any>,
+    options?: CliUx.Table.table.Options
+  ): void;
```

Comment:
Will this ever be called external to the OutputService?  It does not feel like it.  I would remove it from the interface, but keep it in the abstract class.